src/shared/nt-results/ResultsPage.react.js

```diff
@@ -64,13 +65,11 @@ class ResultsPage extends React.Component {
   render () {
     return (
       <div className={styles.nt}>
-        <div className={styles.nt__sidebar}>
-          {this.renderLink('setting')}
-          Sessions
+        <div className={styles.nt__nav}>
+          {this.renderLink('setting', 'cog')}
           {this.props.sessionTypes.map(sessionType => (
-            this.renderLink(url.parse(sessionType.url).path, `/${sessionType._id}`)
+            this.renderLink(url.parse(sessionType.url).path, '', `/${sessionType._id}`)
```

Comment:
Right now this method is duplicate and need to be refactored in the future, so this problem will be gone with the refactoring. Anyways, I'm not sure if it still have time to do this.